odoo/custom/dependencies/pip.txt

```diff
@@ -1,3 +1,5 @@
 git+https://github.com/OCA/openupgradelib.git@master
 unicodecsv
 unidecode
+
+git+https://github.com/Tecnativa/click-odoo-contrib.git@fix-active-modules-hashing
```

Comment:
This is still missing.

Reply:
Ok, I only added it here for testing purposes, if upstream does not answer and we merge this I will move it to the correct place and reference the PR instead of the branch

Reply:
As it already included in Doodba with https://github.com/Tecnativa/doodba/pull/414, I'll just wait for the build and then drop this commit